addons/survey/tests/test_certification_flow.py

```diff
@@ -107,8 +112,12 @@ def test_flow_certification(self):
             self._answer_question(q03, "I think they're great!", answer_token, csrf_token)
             self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token, button_submit='previous')
             self._answer_question(q03, "Just kidding, I don't like it...", answer_token, csrf_token)
-            self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token)
-            self._answer_question(q05, [q05.suggested_answer_ids.ids[0], q05.suggested_answer_ids.ids[1], q05.suggested_answer_ids.ids[3]], answer_token, csrf_token)
+            self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token,
+                                  submit_query_count=49, access_page_query_count=29)
```

Comment:
Ah forgot this one. 
As it's not always a question (after final answer), it could be `access_next_page_query_count`. 
I'll change it at the next iteration if there is one :)